wetterdienst/provider/geosphere/observation/api.py

```diff
@@ -46,63 +46,36 @@ class GeosphereObservationPeriod(Enum):
-            PRESSURE_AIR_SEA_LEVEL = "P0"  # Reduzierter Luftdruck hPa
-            RADIATION_GLOBAL = "GSX"  # Globalstrahlung W/m²
-            RADIATION_SKY_SHORT_WAVE_DIFFUSE = "HSX"  # Himmelsstrahlung/Diffusstrahlung in W/m²
-            SNOW_DEPTH = "SH"  # Gesamtschneehöhe aus Schneepegelmessung cm
-            SUNSHINE_DURATION = "SO"  # Sonnenscheindauer s
-            TEMPERATURE_AIR_MAX_005 = "TSMAX"  # Lufttemperaturmaximum in 5cm °C
-            TEMPERATURE_AIR_MAX_200 = "TLMAX"  # Lufttemperaturmaximum in 2m °C
-            TEMPERATURE_AIR_MEAN_005 = "TS"  # Lufttemperatur in 5cm °C
-            TEMPERATURE_AIR_MEAN_200 = "TL"  # Lufttemperatur in 2m °C
-            TEMPERATURE_AIR_MIN_005 = "TSMIN"  # Lufttemperaturminimum in 5cm °C
-            TEMPERATURE_AIR_MIN_200 = "TLMIN"  # Lufttemperaturminimum in 2m °C
-            TEMPERATURE_DEW_POINT_MEAN_200 = "TP"  # Taupunkt °C
-            TEMPERATURE_SOIL_MEAN_010 = "TB1"  # Erdbodentemperatur in 10cm Tiefe °C
-            TEMPERATURE_SOIL_MEAN_020 = "TB2"  # Erdbodentemperatur in 20cm Tiefe °C
-            TEMPERATURE_SOIL_MEAN_050 = "TB3"  # Erdbodentemperatur in 50cm Tiefe °C
-            WIND_DIRECTION = "DD"  # Windrichtung °
-            WIND_DIRECTION_GUST_MAX = "DDX"  # Windrichtung zum Böenspitzenwert °
-            WIND_GUST_MAX = "FFX"  # maximale Windgeschwindigkeit m/s
-            WIND_SPEED = "FF"  # vektorielle Windgeschwindigkeit m/s
+            PRESSURE_AIR_SEA_LEVEL = "pred"  # Reduzierter Luftdruck hPa
+            RADIATION_GLOBAL = "cglo"  # Globalstrahlung Mittelwert W/m²
+            RADIATION_SKY_SHORT_WAVE_DIFFUSE = "chim"  # Himmelsstrahlung/Diffusstrahlung Mittelwert in W/m²
+            SNOW_DEPTH = "sh"  # Gesamtschneehöhe aus Schneepegelmessung cm
+            SUNSHINE_DURATION = "so"  # Sonnenscheindauer s
+            TEMPERATURE_AIR_MAX_005 = "tsmax"  # Lufttemperaturmaximum in 5cm °C
+            TEMPERATURE_AIR_MAX_200 = "tlmax"  # Lufttemperaturmaximum in 2m °C
+            TEMPERATURE_AIR_MEAN_005 = "ts"  # Lufttemperatur in 5cm °C
+            TEMPERATURE_AIR_MEAN_200 = "tl"  # Lufttemperatur in 2m °C
+            TEMPERATURE_AIR_MIN_005 = "tsmin"  # Lufttemperaturminimum in 5cm °C
+            TEMPERATURE_AIR_MIN_200 = "tsmax"  # Lufttemperaturminimum in 2m °C
+            TEMPERATURE_SOIL_MEAN_010 = "tb10"  # Erdbodentemperatur in 10cm Tiefe °C
+            TEMPERATURE_SOIL_MEAN_020 = "tb20"  # Erdbodentemperatur in 20cm Tiefe °C
+            TEMPERATURE_SOIL_MEAN_050 = "tb50"  # Erdbodentemperatur in 50cm Tiefe °C
+            WIND_DIRECTION = "dd"  # Windrichtung °
+            WIND_DIRECTION_GUST_MAX = "ddx"  # Windrichtung zum Böenspitzenwert °
+            WIND_GUST_MAX = "ffx"  # maximale Windgeschwindigkeit m/s
+            WIND_SPEED = "ff"  # vektorielle Windgeschwindigkeit m/s
+            WIND_SPEED_MEAN_100 = "ffam"  # vektorielle Windgeschwindigkeit in 10m Höhe m/s
+            WIND_SPEED_MAX_TIME = "zeitx"
```

Comment:
Can this parameter be parsed as float?

Reply:
Hi @gutzbenj , I checked that they are already in float when they are read. Is there any particular reason why you'd want me to specifically parse these parameters in float?

Reply:
There's no reason! What I wanted to say is they are going to be parsed as float, so if it can't be parsed as float, the read process will fail.